web/static/css/app.css

```diff
@@ -134,3 +129,8 @@ select:focus {
   border-width: medium;
   border-color: #25bdc3;
 }
+
+.hl-input-error {
+  outline: none !important;
```

Comment:
We seem to have a few `!important` dotted throughout the CSS. Are these definitely necessary?
https://appendto.com/2016/04/css-important-rule-how-to-use-it-correctly/

It's possible that this is causing the need later on (I'm not convinced this is necessary but willing to be told I'm wrong): https://github.com/healthlocker/healthlocker/blob/a2d8c5a8a9c893f591f99fe7d2d6b8442c304cd8/web/static/css/app.css#L16